### DISCUSSION TITLE
Is it possible to connect only to a specific node in a cluster?

### DISCUSSION THREAD

**Discussion by krivulcik** [ravendb/ravendb#12853] [Q&A] [answered]
With some of our larger imports, we run the importing program directly on the server containing one of the cluster nodes.

In earlier versions, we could specify the server to connect to. However, with RavenDB 4 and above, the client detects the cluster topology and connects to a preferred node.

In our use case, it would be helpful to allow the client to connect only to the local node. It would be okay for the program to fail if that's not possible (node down, not healthy, etc.).

We want to decrease latency and the amount of transferred traffic between datacenters by this.

Is something like this currently possible?

**Comment by ayende** [comment, marked as answer]
How are you doing this? Via bulk insert, or the session?

**Reply by krivulcik** [reply]
We need both approaches in different scenarios.

**Reply by ayende** [reply]
In that case, the easiest would be to create a dedicated document store for just that node, like this:

```
var docStore = new DocumentStore
{
    Urls = new[]{"forced.url"},
    Conventions = {DisableTopologyUpdates = true} // don't move away
}

```

**Reply by krivulcik** [reply]
Great, that should do the trick, thanks.
